pocs/tests/test_messaging.py

```diff
@@ -1,21 +1,46 @@
+import multiprocessing
 import pytest
 import time
 
 from datetime import datetime
-from multiprocessing import Process
 from pocs.utils.messaging import PanMessaging
 
 
+@pytest.fixture(scope='module')
```

Comment:
Is this what fixes the issue on travis? Just curious.

Reply:
multiprocessing.Event uses shared memory, while multiprocessing.Manager().Event uses proxies communicating with a shared process.

The scope=module is there just to increase the speed of the tests.

Reply:
Got, thanks.